kubernetes-docs/ss.md

```diff
@@ -0,0 +1,80 @@
+---
+title: StatefulSets
+---
+
+vSphere volumes can be consumed by Stateful Sets
```

Comment:
Can you provide a more better description. A one liner to introduce the whole new section doesn't look very good.